backend/pkg/api/types/dashboard.go

```diff
@@ -5,10 +5,15 @@ type AccountDashboard struct {
 	Name string `json:"name"`
 }
 type ValidatorDashboard struct {
-	Id        uint64        `json:"id"`
-	Name      string        `json:"name"`
-	PublicIds []VDBPublicId `json:"public_ids,omitempty"`
+	Id             uint64        `json:"id"`
+	Name           string        `json:"name"`
+	PublicIds      []VDBPublicId `json:"public_ids,omitempty"`
+	IsArchived     bool          `json:"is_archived"`
+	ArchivedReason string        `json:"archived_reason,omitempty"` // dashboard_limit, validator_limit, group_limit, none
```

Comment:
Frontend would probably like this:
```suggestion
	ArchivedReason string        `json:"archived_reason,omitempty" tstype:"'dashboard_limit' | 'validator_limit' | 'group_limit'"`
```
Don't forget to convert to ts